### ISSUE TITLE
Implement List tokens UI

### ISSUE THREAD

**Issue by laushinka** [gitpod-io/gitpod#14860]
Show the list view as in the designs (without creation functionality).

- [Link](https://gitpod.slack.com/archives/C01KPEPQYE6/p1669129212853829) to the empty design
- [Link](https://gitpod.slack.com/archives/C01KPEPQYE6/p1669151203668149) to the view with items

| [Empty](https://gitpod.slack.com/archives/C01KPEPQYE6/p1669129212853829) | [Has items](https://gitpod.slack.com/archives/C01KPEPQYE6/p1669151203668149) |
| - | - |
| <img width="1440" alt="Empty" src="https://user-images.githubusercontent.com/120486/203577367-bfd2aa0e-7f4b-4659-92de-a94f88caf794.png"> | <img width="1440" alt="Regenerated" src="https://user-images.githubusercontent.com/120486/203577496-ce595913-b763-415e-8be7-ad1b08fdf688.png"> |

**Comment by laushinka** [comment]
Tagging you here @gtsiolis for visibility. I will be working on this.

**Comment by laushinka** [comment]
@gtsiolis Did we have an idea what the `Learn more` link would redirect to?

**Comment by gtsiolis** [comment]
@laushinka I was hoping for a section in our **docs**[[1](https://www.gitpod.io/docs/configure/user-settings)], but feel free to drop the link if we are not planning to add docs before adding this feature. Cc @easyCZ @jldec

**Comment by laushinka** [comment]
> @laushinka I was hoping for a section in our **docs**

Good point 💯 By the time this gets rolled out we should have documentation for it as well. I'll create a separate issue for this (instead of including a placeholder link now) so we don't forget. Thanks!

**Comment by mustard-mh** [comment]
@laushinka  Does `List tokens` include create?

**Comment by laushinka** [comment]
> @laushinka Does list tokens include create?

@mustard-mh Hi, Huiwen! It doesn't include create as I wrote in the description. 

> Show the list view as in the designs (without creation functionality).

Will you be working on the create functionality or other UI? If yes let's sync so I can help unblock or align :)

**Comment by gtsiolis** [comment]
**fyi**: Updated the designs in the issue description to include the latest which do not include the _Last used_ element.
